add_up.rb

```diff
@@ -0,0 +1,19 @@
+## recursive
+# def add_up(i)
+#   if i > 0
+#     i + add_up(i-1)
+#   else
+#     0
+#   end
+# end
```

Comment:
👏 👏 👏 Very impressed with attempting a recursive solution (and succeeding!)

Reply:
Thank you. I comment it out because recursion is not as straightforward using the `downto` method or the `.sum` for range.